packages/core/src/hooks/useTransition.test.tsx

```diff
@@ -35,6 +36,17 @@ describe('useTransition', () => {
     expect(rendered).toEqual([false])
   })
 
+  it('assign controller to provided "ref"', async () => {
```

Comment:
Can we make a more realistic test in the case of `useTransition` i.e. typically we might have 3 items in our Transition, so in my head, the ref should have three `controller`s?

Reply:
:+1: good idea -> updated